cea/demand/ventilation_model.py

```diff
@@ -409,7 +409,7 @@ def allocate_default_leakage_paths(coeff_lea_zone, area_facade_zone, area_roof_z
 
     coeff_lea_path = np.zeros(5)
     height_lea_path = np.zeros(5)
-    orientation_lea_path = np.zeros(5)
+    orientation_lea_path = [0, 1, 0, 1, 2]
```

Comment:
The reason has to do with data types. `np.zeroes` returns an array of floats, but `orientation_lea_path` is actually used to retrieve an index into another array. Indexes need to be integers. I was getting some errors while debugging because they were floats.

BTW: once you have indexes into arrays to retrieve array indexes into other arrays, that can be considered a code smell and yes, here would be a good place to sprinkle some objects...